apps/app/src/client/components/Admin/Security/SecuritySetting.jsx

```diff
@@ -368,11 +368,36 @@ class SecuritySetting extends React.Component {
             <tbody>
               <tr>
                 <th scope="row">{ t('public') }</th>
-                <td><span className="material-symbols-outlined text-success me-1">check_circle</span>{ t('security_settings.always_displayed') }</td>
+                <td>
+                  <div className="form-check form-switch form-check-success">
+                    <input
+                      type="checkbox"
+                      disabled
+                      className="form-check-input"
+                      id="isShowPublic"
+                      checked
+                    />
+                    <label className="form-label form-check-label" htmlFor="isShowPublic">
+                      {t('security_settings.fixed')}
+                    </label>
+                  </div>
+                </td>
               </tr>
               <tr>
                 <th scope="row">{ t('anyone_with_the_link') }</th>
-                <td><span className="material-symbols-outlined text-danger me-1">cancel</span>{ t('security_settings.always_hidden') }</td>
+                <td>
+                  <div className="form-check form-switch form-check-success">
+                    <input
+                      type="checkbox"
+                      disabled
+                      className="form-check-input"
+                      id="isShowRestrictedByAnyonewiththelink"
```

Comment:
```suggestion
                      id="isShowRestrictedByAnyoneWithTheLink"
```

キャメルケースにする
参考: https://developer.mozilla.org/ja/docs/Glossary/Camel_case